satpy/etc/readers/ahi_hsd.yaml

```diff
@@ -289,64 +289,80 @@ file_types:
   hsd_b01:
     file_reader: !!python/name:satpy.readers.ahi_hsd.AHIHSDFileHandler
     file_patterns: ['HS_{platform_shortname}_{start_time:%Y%m%d_%H%M}_B01_{area}_R10_S{segment:2d}{total_segments:2d}.DAT',
+                    'HS_{platform_shortname}_{start_time:%Y%m%d_%H%M}_B01_{area}_R20_S{segment:2d}{total_segments:2d}.DAT',
```

Comment:
Question: Would it be worth it to update the first pattern so `R10` becomes `R{res_id:d}` or something like that? That way we still only have 2 total file patterns..

Reply:
This thought did cross my mind as I was doubling up on the code.

Reply:
I've added this instead, makes sense RE code double up.